src/components/Search/index.js

```diff
@@ -1,19 +1,37 @@
 import React from 'react';
 import Switch from '../Switch';
+import { Link } from 'react-router-dom'
 
 import './style.css';
 
 class Search extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            searchString: '',
+        };
+    }
+
+    updateSearchString(e) {
+        this.setState({
+            searchString: e.target.value
+        });
+    }
+
     render() {
         return (
             <div className="search">
                 <div className="search__header">
                     netflixroulette
                 </div>
                 <div className="search__search-controls">
-                    <input className="search_input" />
+                    <input className="search_input" value={this.state.searchString} onChange={e => this.updateSearchString(e)} />
```

Comment:
What was the reason to use arrow function here?
Why not to replace it with onChange={this.updateSearchString} and do this.updateSearchString = this.updateSearchString.bind(this); in the constructor?